src/mt19937-64.c

```diff
@@ -61,73 +61,79 @@
 #define MM 156
 #define MATRIX_A 0xB5026F5AA96619E9ULL
 #define UM 0xFFFFFFFF80000000ULL /* Most significant 33 bits */
-#define LM 0x7FFFFFFFULL /* Least significant 31 bits */
+#define LM 0x7FFFFFFFULL         /* Least significant 31 bits */
```

Comment:
We should exclude the files that were copied instead of written for the project. This will just make future merges annoying. I think the list is mt19937-64.c/h, lfz_c, sha256, sha1, strl.c

Reply:
And pqsort.c, crccombine.*, crcspeed.*

We can put them in a file called `.clang-format-ignore`

Reply:
Or we can add the do-not-format annotation to the files, marking them from start to end.  Then if we have to patch them, we can use multiple annotations so that we still autoformat our patches.